current/aut-spark-submit-app.md

```diff
@@ -66,18 +66,30 @@ following columns: `crawl_date`, `src_domain`, `dest_domain`, and `count`.
 columns are an array:
 `((CrawlDate, SourceDomain, DestinationDomain), Frequency)`
 
+Text output:
+
+```shell
+spark-submit --class io.archivesunleashed.app.CommandLineAppRunner path/to/aut-fatjar.jar --extractor ImageGraphExtractor --input /path/to/warcs/* --output output/path --output-format TEXT
```

Comment:
I couldn't get this to work without the `--df` flag (i.e. the commands below worked). Running the above command leads to:

```
20/04/08 16:09:28 ERROR CommandLineApp: ImageGraphExtractor not supported with RDD. The following extractors are supported:
20/04/08 16:09:28 ERROR CommandLineApp: DomainFrequencyExtractor
20/04/08 16:09:28 ERROR CommandLineApp: DomainGraphExtractor
20/04/08 16:09:28 ERROR CommandLineApp: PlainTextExtractor
```